src/configuration/services/mysql.md

```diff
@@ -12,10 +12,12 @@ The service types `mariadb` and `mysql` both refer to MariaDB for compatibility
 * mariadb:10.1
 * mariadb:10.2
 
+
```

Comment:
Why the extra line break here?  Seems unnecessary.  One should be sufficient.

Reply:
For posterity: Because GitBook is breaking the list formatting otherwise and making it like 5 different lists, because reasons.